face/capture/src/test/java/com/simprints/face/capture/screens/FaceCaptureViewModelTest.kt

```diff
@@ -302,6 +307,30 @@ class FaceCaptureViewModelTest {
         assertThat(viewModel.isAutoCaptureEnabled.getOrAwaitValue()).isFalse()
     }
 
+    @Test
+    fun `preparation instructions screen should be set to showing according to its use case`() {
+        // Given
+        coEvery { getAndTurnOffPreparationInstructionsShowing() } returns true
+
+        // When
+        val isShowing = viewModel.getAndTurnOffIfPreparationInstructionsShowing()
+
+        // Then
+        assertThat(isShowing).isTrue()
+    }
+
+    @Test
+    fun `preparation instructions screen should be set to not showing according to its use case`() {
```

Comment:
Considering that the method just does the deligation to the use case, I could argue that this test case is redundant.